SDK/AppCenter/Microsoft.AppCenter.UWP/Utils/DeviceInformationHelper.cs

```diff
@@ -16,20 +16,35 @@ public class DeviceInformationHelper : AbstractDeviceInformationHelper
     {
         public static event EventHandler InformationInvalidated;
         private static string _country;
-        private readonly IScreenSizeProvider _screenSizeProvider;
-        private static IScreenSizeProviderFactory _screenSizeProviderFactory =
+        private static IScreenSizeProvider ScreenSizeProvider;
+        private static IScreenSizeProviderFactory ScreenSizeProviderFactory =
```

Comment:
Naming convention for `private static` property should be `_screenSizeProvider ` and `_screenSizeProviderFactory`.